api/nomics.py

```diff
@@ -21,9 +21,9 @@ def info():
                        " products by means of on-chain voting. It is also the first DPoS blockchain in existence and"
                        " the first blockchain to implement stablecoins.",
         "location": "Worldwide",
-        "logo": "https://bitshares.org/exchange-logo.png",
+        "logo": "https://www.bitshares.foundation/download/bitshares-logo.svg",
         "website": "https://bitshares.org/",
```

Comment:
I am not sure if the logo will not work, check specs:

https://github.com/nomics-crypto/nomics-platform/blob/master/doc/cryptocurrency-api-exchange-integration.md#response

Apparently it needs to be square.

Reply:
Fixed

https://www.bitshares.foundation/download/bitshares-logo-squared.svg